src/translate_c.zig

```diff
@@ -130,7 +130,7 @@ const Scope = struct {
         fn makeMangledName(scope: *Block, c: *Context, name: []const u8) ![]const u8 {
             const name_copy = try c.arena.dupe(u8, name);
             var proposed_name = name_copy;
-            while (scope.contains(proposed_name)) {
+            while (scope.contains(proposed_name) or mem.eql(u8, proposed_name, "_")) {
```

Comment:
Why is this check needed? `@"_"` should work like any other identifier and not need special casing.

Reply:
Probably a stage 1 issue, but if a local variable is named `_` I get a compile error in the translated zig:

```
./test.zig:1027:5: error: `_` is not a declarable symbol
    var @"_": c_int = 42;
    ^
```

Reply:
Simply removing that declarable symbol check from astgen.cpp is not sufficient; doing so results in this:

```zig
pub export fn bar(arg_x: c_int) void {
    var x = arg_x;
    var @"_": c_int = 42;
    if (@"_" != x) {
        abort();
    }
}
```

```
./test.zig:1028:9: error: use of undeclared identifier '_'
    if (@"_" != x) {
        ^
```

Reply:
I see, I think I'd still prefer to remove that check and leave that in as a stage1 miscompilation as it doesn't cause any `ast-check` errors.

Reply:
Ok, removed it and took out the relevant test code